qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -199,15 +198,14 @@ def write_local_config(self):
         self.test_dev = mon_sock + ':/' + self.test_dirname
         self.scratch_dev = mon_sock + ':/' + self.scratch_dirname
 
-        xfstests_config_contents = dedent('''\
```

Comment:
this looks to be just using the formation string. SCRATCH_MNT and SCRATCH_DEV aren't being changed as mentioned in the commit message, isn't it?

Reply:
Right. Xiubo enabled these options and that change was absorbed by this PR during rebase. Commit message needs to be updated.

Reply:
Sure.

Reply:
I've updated the commit message accordingly.